src/RefreshOptions.ts

```diff
@@ -25,3 +25,17 @@ export interface RefreshOptions {
      */
     watchedSettings?: WatchedSetting[];
 }
+
+export interface FeatureFlagRefreshOptions {
+    /**
+     * Specifies whether the provider should automatically refresh when the configuration is changed.
+     */
+    enabled: boolean;
+
+    /**
+     * Specifies the minimum time that must elapse before checking the server for any new changes.
+     * Default value is 10 seconds. Must be greater than 1 second.

```

Comment:
Default should be 30 seconds, right?

Reply:
Kv is 30s, this 10s is referred from k8s provide. I shall double confirm that. So in dotnet it’s 30s right?

Reply:
updated in e269596